ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py

```diff
@@ -329,3 +329,31 @@ def test_numpy_diff(
         x=x[0],
         axis=axis,
     )
+    
+    
+# ediff1d
+@handle_frontend_test(
+    fn_tree="numpy.ediff1d",
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=3
```

Comment:
Based on the error I see in the CI test, I think you'll need to set the minimum number of dimensions to 1.

Reply:
Thank you for checking it out. Appreciate it.

Reply:
Also min_dim_size should be kept as 1